labs/scorex.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: ScoreX
+parent: Labs
+---
+# Lab Name
+ScoreX
+
+# Short Description
+
+ScoreX is a modular blockchain framework built with flexibility in mind to accelerate the design of experimental DTL technologies. ScoreX is the hyperflexible answer to the Fabric and Sawtooth frameworks. ScoreX is developed and maintained by IOHK and the founders of Ergo under the CC0 1.0 license.
```

Comment:
[Hyperledger's charter](https://hyperledger.org/about/charter) has the following requirements regarding licensing:

> 12. Intellectual Property Policy
> 
> a. Members agree that all new inbound code contributions to HLP shall be made under the Apache License, Version 2.0 (available at http://www.apache.org/licenses/LICENSE-2.0). All contributions shall be accompanied by a Developer Certificate of Origin sign-off (http://developercertificate.org) that is submitted through a Governing Board and LF-approved contribution process. Such contribution process will include steps to also bind non-Member Contributors and, if not self-employed, their employer, to the licenses expressly granted in the Apache License, Version 2.0 with respect to such contribution.
> 
> b. All outbound code will be made available under the Apache License, Version 2.0.
> 
> c. All documentation will be contributed to and made available by HLP under the Creative Commons Attribution 4.0 International License (available at http://creativecommons.org/licenses/by/4.0/).
> 
> d. If an alternative inbound or outbound license is required for compliance with the license for a leveraged open source project or is otherwise required to achieve HLP’s mission, the Governing Board may approve the use of an alternative license for specific inbound or outbound contributions on an exception basis. Any exceptions must be approved by a two-thirds vote of the entire Governing Board and the LF and must be limited in scope to what is required for such purpose. Please email legal@hyperledger.org to obtain exception approval.
> 
> e. Subject to available Project funds, HLP may engage The Linux Foundation to determine the availability of, and register, trademarks, service marks, and certification marks, which shall be owned by the LF.

Reply:
Two pieces of potential concern:
1. Source code license in Hyperledger is Apache 2.0. Are you willing to change the license?
2. Is ScoreX a trademark that is already owned by Ergo?

Reply:
@tkuhrt 
1) Yes, we can discuss license change
2) Trademark is not owned by any party

Reply:
Let us know when you have discussed the license change and your decision, @kushti.

Reply:
License change PR is up https://github.com/ScorexFoundation/Scorex/pull/385

Reply:
I am going to mark this as unresolved until the PR is merged.

Reply:
@tkuhrt merged!